File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/phys/PHY2001U04GrdInpInfo.java
package nta.med.data.model.ihis.phys;

public class PHY2001U04GrdInpInfo {
	private String orderDate       ;
	private String bunho            ;
	private String suname           ;
	private String suname2          ;
	private String doctorName      ;
	private String hangmogCode     ;
	private String hangmogName     ;
	private String ptFlag          ;
	private String otFlag          ;
	private String stFlag          ;
	private String buFlag          ;
	private String ocsFlag         ;
	public PHY2001U04GrdInpInfo(String orderDate, String bunho, String suname,
			String suname2, String doctorName, String hangmogCode,
			String hangmogName, String ptFlag, String otFlag, String stFlag,
			String buFlag, String ocsFlag) {
		super();
		this.orderDate = orderDate;
		this.bunho = bunho;
		this.suname = suname;
		this.suname2 = suname2;
		this.doctorName = doctorName;
		this.hangmogCode = hangmogCode;
		this.hangmogName = hangmogName;
		this.ptFlag = ptFlag;
		this.otFlag = otFlag;
		this.stFlag = stFlag;
		this.buFlag = buFlag;
		this.ocsFlag = ocsFlag;
	}
	public String getOrderDate() {
		return orderDate;
	}
	public void setOrderDate(String orderDate) {
		this.orderDate = orderDate;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getSuname() {
		return suname;
	}
	public void setSuname(String suname) {
		this.suname = suname;
	}
	public String getSuname2() {
		return suname2;
	}
	public void setSuname2(String suname2) {
		this.suname2 = suname2;
	}
	public String getDoctorName() {
		return doctorName;
	}
	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}
	public String getHangmogCode() {
		return hangmogCode;
	}
	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}
	public String getHangmogName() {
		return hangmogName;
	}
	public void setHangmogName(String hangmogName) {
		this.hangmogName = hangmogName;
	}
	public String getPtFlag() {
		return ptFlag;
	}
	public void setPtFlag(String ptFlag) {
		this.ptFlag = ptFlag;
	}
	public String getOtFlag() {
		return otFlag;
	}
	public void setOtFlag(String otFlag) {
		this.otFlag = otFlag;
	}
	public String getStFlag() {
		return stFlag;
	}
	public void setStFlag(String stFlag) {
		this.stFlag = stFlag;
	}
	public String getBuFlag() {
		return buFlag;
	}
	public void setBuFlag(String buFlag) {
		this.buFlag = buFlag;
	}
	public String getOcsFlag() {
		return ocsFlag;
	}
	public void setOcsFlag(String ocsFlag) {
		this.ocsFlag = ocsFlag;
	}
	

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Phys/PHY2001U04GrdInpInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Phys
{
    [Serializable]
    public class PHY2001U04GrdInpInfo
    {
        private String _orderDate;
        private String _bunho;
        private String _suname;
        private String _suname2;
        private String _doctorName;
        private String _hangmogCode;
        private String _hangmogName;
        private String _ptFlag;
        private String _otFlag;
        private String _stFlag;
        private String _buFlag;
        private String _ocsFlag;

        public String OrderDate
        {
            get { return this._orderDate; }
            set { this._orderDate = value; }
        }

        public String Bunho
        {
            get { return this._bunho; }
            set { this._bunho = value; }
        }

        public String Suname
        {
            get { return this._suname; }
            set { this._suname = value; }
        }

        public String Suname2
        {
            get { return this._suname2; }
            set { this._suname2 = value; }
        }

        public String DoctorName
        {
            get { return this._doctorName; }
            set { this._doctorName = value; }
        }

        public String HangmogCode
        {
            get { return this._hangmogCode; }
            set { this._hangmogCode = value; }
        }

        public String HangmogName
        {
            get { return this._hangmogName; }
            set { this._hangmogName = value; }
        }

        public String PtFlag
        {
            get { return this._ptFlag; }
            set { this._ptFlag = value; }
        }

        public String OtFlag
        {
            get { return this._otFlag; }
            set { this._otFlag = value; }
        }

        public String StFlag
        {
            get { return this._stFlag; }
            set { this._stFlag = value; }
        }

        public String BuFlag
        {
            get { return this._buFlag; }
            set { this._buFlag = value; }
        }

        public String OcsFlag
        {
            get { return this._ocsFlag; }
            set { this._ocsFlag = value; }
        }

        public PHY2001U04GrdInpInfo() { }

        public PHY2001U04GrdInpInfo(String orderDate, String bunho, String suname, String suname2, String doctorName, String hangmogCode, String hangmogName, String ptFlag, String otFlag, String stFlag, String buFlag, String ocsFlag)
        {
            this._orderDate = orderDate;
            this._bunho = bunho;
            this._suname = suname;
            this._suname2 = suname2;
            this._doctorName = doctorName;
            this._hangmogCode = hangmogCode;
            this._hangmogName = hangmogName;
            this._ptFlag = ptFlag;
            this._otFlag = otFlag;
            this._stFlag = stFlag;
            this._buFlag = buFlag;
            this._ocsFlag = ocsFlag;
        }

    }
}